app/views/spree/admin/orders/bulk_management.html.haml

```diff
@@ -51,47 +51,44 @@
   %hr.divider.sixteen.columns.alpha.omega{ ng: { show: 'unitsVariantSelected()' } }
 
   %div.sixteen.columns.alpha.omega#group_buy_calculation{ ng: { show: 'unitsVariantSelected()', cloak: true } }
-    %div.shared_resource{ :class => "four columns alpha" }
-      %span{ :class => 'three columns alpha' }
-        %input{ type: 'checkbox', :id => 'shared_resource', 'ng-model' => 'sharedResource'}
+    .one.columns.alpha
+    .shared_resource.four.columns.alpha
+      %input{ type: 'checkbox', :id => 'shared_resource', 'ng-model' => 'sharedResource'}
+      %span
         = t("admin.orders.bulk_management.shared")
-    %div{ :class => "eight columns" }
-      %h6{ :class => "eight columns alpha", 'ng-show' => 'sharedResource', style: 'text-align: center;' } {{ selectedUnitsProduct.name + ": ALL" }}
-      %h6{ :class => "eight columns alpha", 'ng-hide' => 'sharedResource', style: 'text-align: center;' } {{ selectedUnitsVariant.full_name }}
-    %div{ :class => "four columns omega" }
-      %h6{ :class => "four columns alpha", :style => 'text-align: right;' }
+    .eight.columns
+      %h6.text-center{ 'ng-show' => 'sharedResource' } {{ selectedUnitsProduct.name + ": ALL" }}
```

Comment:
Is the `ALL` at the end of this line a string that should be translated?

Reply:
Yes, I suppose: 
<img width="842" alt="Capture d’écran 2021-02-15 à 09 09 25" src="https://user-images.githubusercontent.com/296452/107920598-877a7500-6f6d-11eb-8562-e03917b03f77.png">

To resolve this should I open an issue, or correct into this PR (despite the fact that it's unrelated to my PR)?

Reply:
I'd put it as part of this PR, since it looks like it adds the "ALL" string.

Reply:
The PR doesn't add this, previous line was too long to fit into one line.
<img width="883" alt="Capture d’écran 2021-02-15 à 22 05 58" src="https://user-images.githubusercontent.com/296452/107992991-3359aa00-6fda-11eb-9060-664e5dd49625.png">

Reply:
Oh, got it, sorry. Then yeah - up to you, if it's easier to just include it here, then I'd go that route, otherwise fine to create an issue (and probably add the good-first-issue label)

Reply:
Created a new issue: #6892

Reply:
Yeah it's not part of the changes in the PR, I just noticed it while reviewing :ok_hand: